src/fparser/tests/test_blank_lines.py

```diff
@@ -104,4 +108,9 @@ def test_reproduce_issue():
         SUBROUTINE blc(a, b, c, d, e)
           a = b + c + d + e
         END SUBROUTINE blc
-    '''.strip().split('\n')[1:]
+    """.strip().split(
```

Comment:
I looked at the resulting code and I like the results of most of it, but there is this case when we have:
```
"""
multipline string
""".strip().split(\n)[1:]
```
which gets converted to:
```
   """.strip().split(
           "\n"
       )[
           1:
       ]
   )
```
because since it already detects a multiline statement it keeps splitting.

Could we convert this to something like:
```
expected = """
multiline string
"""
expected = expected.strip().split("\n")[1:]
assert str(tree).strip().split("\n")[1:] == expected

```
So applying black doesn't split the lines?

There are a few of this, feel free to tell me if you think the blacked version is already alright.

Reply:
Oh yes, it's much nicer if we make the change you suggest. I'll try and find those cases and fix them.